### PR TITLE
Typo and grammar fix

### PR DESCRIPTION
**Pull request by VaibhavWakde52**

### CODE REVIEW

**Comment by VaibhavWakde52** [comment]
I have read the Mondoo CLA Document and I hereby sign the CLA

**Comment by VaibhavWakde52** [comment]
recheck

File: docs/development.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ outlined above.
 
 ## Debug providers
 
-In v9 we introduced providers, which split up the providers into individual go modules. This make it more development
+In v9 we introduced providers, which split up the providers into individual go modules. This makes it more development
```
Comment (misterpantz) [inline docs/development.md]:
```suggestion
In v9 we introduced provider plugins, which split up the providers into individual go modules. This makes development 
```

**Review by tas50 — APPROVED** [review]

**Review by misterpantz — APPROVED** [review]
Thank you, @VaibhavWakde52!

**Pull request merged**